asistente_ladm_col/gui/dialog_input_load_field_data_capture_dialog.py

```diff
@@ -0,0 +1,480 @@
+            FDC_ADMINISTRATIVE_SOURCE: {'name': FDC_ADMINISTRATIVE_SOURCE, 'geometry': None},
+            FDC_RRRSOURCE: {'name': FDC_RRRSOURCE, 'geometry': None},
+            FDC_UEBAUNIT: {'name': FDC_UEBAUNIT, 'geometry': None}
+        }, load=True)
+
+    def load_gdb(self):
+        self.txt_log.setText(QCoreApplication.translate("DialogInputLoadFieldDataCapture", "Loading GDB tables..."))
+        gdb_path = self.txt_file_path_gdb.text()
+        layer = QgsVectorLayer(gdb_path, 'layer name', 'ogr')
+        info = layer.dataProvider().subLayers()
+
+        root = QgsProject.instance().layerTreeRoot()
+        gdb_group = root.addGroup("GDB")
+        self.gdb_layer = []
+
+        for data in info:
+            vlayer = QgsVectorLayer(gdb_path + '|layername=' + data.split('!!::!!')[1], data.split('!!::!!')[1], 'ogr')
+            self.gdb_layer.append(vlayer)
+            QgsProject.instance().addMapLayer(vlayer, False)
+            gdb_group.addLayer(vlayer)
+
+        self.res_layers_gdb = self.qgis_utils.get_layers(self._db, {
+            FDC_PLOT: {'name': FDC_PLOT, 'geometry': QgsWkbTypes.PolygonGeometry},
+            FDC_SECTOR: {'name': FDC_SECTOR, 'geometry': None},
+            FDC_VILLAGE: {'name': FDC_VILLAGE, 'geometry': None},
+            FDC_NEIGHBOURHOOD: {'name': FDC_NEIGHBOURHOOD, 'geometry': None},
+            FDC_BUILDING: {'name': FDC_BUILDING, 'geometry': QgsWkbTypes.PolygonGeometry},
+            FDC_VALUATION_BUILDING: {'name': FDC_VALUATION_BUILDING, 'geometry': None},
+            FDC_BUILDING_UNIT_VALUATION_TABLE: {'name': FDC_BUILDING_UNIT_VALUATION_TABLE, 'geometry': None},
+            FDC_BUILDING_UNIT_CADASTRE_TABLE: {'name': FDC_BUILDING_UNIT_CADASTRE_TABLE, 'geometry': QgsWkbTypes.PolygonGeometry},
+            FDC_VALUATION_UNIT_BUILDING_CONNECTION: {'name': FDC_VALUATION_UNIT_BUILDING_CONNECTION, 'geometry': None},
+            FDC_VALUATION_BUILDING_CONNECTION: {'name': FDC_VALUATION_BUILDING_CONNECTION, 'geometry': None},
+            FDC_QUALIFICATION_CONVENTIONAL: {'name': FDC_QUALIFICATION_CONVENTIONAL, 'geometry': None},
+            FDC_QUALIFICATION_NO_CONVENTIONAL: {'name': FDC_QUALIFICATION_NO_CONVENTIONAL, 'geometry': None},
+            FDC_EXTADDRESS: {'name': FDC_EXTADDRESS, 'geometry': QgsWkbTypes.PointGeometry}
+        }, load=True)
+
+    def mapping_fields_r1(self):
+        steps = 29
+        step = 0
+
+        self.txt_log.setText(QCoreApplication.translate("DialogInputLoadFieldDataCapture", "Loading data to {} table...".format(FDC_PARCEL)))
+        query = "select * from {} group by NoPredial&nogeometry".format(self.layer_r1.name())
+        input_data = QgsVectorLayer( "?query={}".format(query), "vlayer", "virtual")
+        create_column(self.res_layers[FDC_PARCEL], 'Codigo')
+        run_etl_model_input_load_data(input_data, self.res_layers[FDC_PARCEL], FDC_PARCEL, self.qgis_utils)
+        step += 1
+        self.progress.setValue(step/steps * 100)
```

Comment:
After updating the progressbar please add a blank line and a comment introducing the next step.

Reply:
Thanks, solved